src/ast/mod.rs

```diff
@@ -4304,6 +4368,39 @@ impl fmt::Display for CreateFunctionUsing {
     }
 }
 
+/// DuckDB specific feature.
+///
+/// See [Create Macro - DuckDB](https://duckdb.org/docs/sql/statements/create_macro)
+/// for more details
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub struct OperateMacroArg {
```

Comment:
What is the meaning of the `Operate` prefix? Maybe this could be shortened to `MacroArg`?

Reply:
Got it from OperateFunctionArg.